developers/compute-to-data/README.md

```diff
@@ -20,19 +20,6 @@ The Ocean Protocol provides a compute environment that you can access at the fol
 
 We suggest reading these guides to get an understanding of how compute-to-data works:
 
-### Architecture & Overview Guides
```

Comment:
These need to be updated, not deleted, I have a PR for these https://github.com/oceanprotocol/docs/pull/1513